lib/variant-model.js

```diff
@@ -45,17 +47,47 @@ _.extend(Variant.prototype, {
     return this;
   },
 
-  respondWithFile: function(fileName) {
+  respondWithFile: function(options) {
+    options = _.extend({
+      variant_id: this.id(),
+      path: this._route.path(),
+    }, options);
+
     return this.respondWith(function(request, reply) {
-      var self = this;
-      fileName = fileName.replace(/\{([^\}]+)\}/g, function(match, token) {
-        var val = request.params[token];
-        if (!val) {
-          val = self.state(token);
-        }
-        return val || match;
-      });
-      reply.file(fileName);
+      var smocks = require('./index');
+
+      options = _.extend({
+        fileName: smocks.initOptions.fileName || 'response.json',
+        mockDir: smocks.initOptions.mockDir || './mocked-data',
+        statusCode: smocks.initOptions.statusCode || 200
+      }, options);
+
+      if(options.statusCode == 200) {
```

Comment:
Added this temporary.

reply.file(dataCommonFilePath).code(234); 
This doesn't work for reply.file() only reply("fdsfdfsf").code(234), any ideas?